rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionCommand.java

```diff
@@ -12,7 +12,14 @@
 
     public SolutionCommand( CommandAction... actions )
     {
-        this.actions = actions;
+        if ( 0 == actions.length )
+        {
+            this.actions = new CommandAction[] { new WaitAction( 1 ) };
+        }
+        else
+        {
+            this.actions = actions;
+        }
     }
```

Comment:
Couldn't we pass on an array of size 1? I don't feel like that knowledge should live here.